### ISSUE TITLE
Release a Premium offering

### ISSUE THREAD

**Issue by neomodern** [turtl/js#219]
Your app has a lot of potential but development seems to be going slow. I think you should hurry up and release some form of premium offering - it doesn't have to be anything imaginative, even just an increased storage limit is _something_ which supporters subscribe to and it will give both motivation + money to fuel further development.

**Comment by Lcstyle** [comment]
agreed, I think you will find many people want to jump ship from other insecure products that are charging AND data mining private user data.

**Comment by marioblokland** [comment]
+1

**Comment by Lcstyle** [comment]
don't forget to add an evernote migration tool so we can make the switch when you release your premium offering!
